src/lib/repositories/impl/order_repository_impl.py

```diff
@@ -25,3 +26,11 @@ def delete_by_id(self, order_id):
     def update_by_id(self, order_id, order):
         current_order = self.get_by_id(order_id)
         current_order.order_details = order.order_details or current_order.order_details
+
+    def get_orders_to_process(self):
+        orders = self.get_all()
+
+        orders_to_process = [
```

Comment:
I think there are built-in functions in python to apply filters in lists: https://www.geeksforgeeks.org/filter-in-python/

Reply:
done. i used filter() + lambda function.